frontend/src/api/trace/getTraceItem.ts

```diff
@@ -2,14 +2,14 @@ import axios from 'api';
 import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
 import { AxiosError } from 'axios';
 import { ErrorResponse, SuccessResponse } from 'types/api';
-import { PayloadProps, Props } from 'types/api/trace/getTraceItem';
+import { PayloadProps, GetTraceItemProps } from 'types/api/trace/getTraceItem';
 
 const getTraceItem = async (
-	props: Props,
+	props: GetTraceItemProps,
 ): Promise<SuccessResponse<PayloadProps> | ErrorResponse> => {
 	try {
 		const response = await axios.request<PayloadProps>({
-			url: `/traces/${props.id}`,
+			url: `/traces/${props.id}?spanId=${props.spanId || ''}&levelUp=${props.levelUp || ''}&levelDown=${props.levelDown || ''}`,
```

Comment:
please make util for this one ... if value is present then only append in the string else not